src/app/login-page/login-page.component.scss

```diff
@@ -0,0 +1,38 @@
+.loginCont{
+  display: flex;
+  flex-flow: column;
+  width: 400px;
+  height: 295px;
+  margin: 0 auto;
+  border: 1px solid black;
+  background-color: rgb(238, 238, 238);
+  align-items: center;
+
+}
+
+.inputField{
+  width: 60%;
+  height: 48px;
+  margin: 5px;
+  min-width: 200px;
+  border: 1px solid black;
+}
+
+.interactionsCont{
+  display: flex;
+  flex-flow: row;
+  width: 60%;
+  justify-content: space-between;
+}
+
+.loginButton{
+  width: 113px;
+  height: 38px;
+  background-color: rgb(189, 214, 172);
+  border: 1px solid black;
+}
+
+.messageField{
+  color: red;
+}
+
```

Comment:
Give placeholders black color instead of grey.

Reply:
Left this as is as discussed, as placeholder shouldn't be the same colour as the input text, which is black.